simanneal/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "simanneal" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0657354bacd503ff5fc0b984110be7b174ecedbb976caf1c230c73a128a06d5a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - simanneal
+
+about:
+  home: https://github.com/perrygeo/simanneal
+  license: ISC
+  license_family: BSD
```

Comment:
You need to have a license file entry as seen here: https://conda-forge.org/docs/meta.html#packaging-the-license-manually